besu/src/main/java/org/hyperledger/besu/services/PicoCLIOptionsImpl.java

```diff
@@ -28,6 +34,24 @@ public PicoCLIOptionsImpl(final CommandLine commandLine) {
 
   @Override
   public void addPicoCLIOptions(final String namespace, final Object optionObject) {
-    commandLine.addMixin("Plugin " + namespace, optionObject);
+    final String pluginPrefix = "--plugin-" + namespace + "-";
+    final String unstablePrefix = "--Xplugin-" + namespace + "-";
+    final CommandSpec mixin = CommandSpec.forAnnotatedObject(optionObject);
+    boolean badOptionName = false;
+
+    for (final OptionSpec optionSpec : mixin.options()) {
+      for (final String optionName : optionSpec.names()) {
+        if (!optionName.startsWith(pluginPrefix) && !optionName.startsWith(unstablePrefix)) {
+          badOptionName = true;
```

Comment:
Don't you think we should fail fast here ?

Reply:
I want to enumerate all of the bad options, not just the first I find.